superbench/benchmarks/micro_benchmarks/ib_validation_performance.py

```diff
@@ -6,7 +6,7 @@
 import os
 
 from superbench.common.utils import logger
-from superbench.common.utils import gen_topo_aware_config
+from superbench.common.utils import gen_topo_aware_config, gen_pair_wise_config
```

Comment:
I think it should be separated PR.

Reply:
separated this part to https://github.com/microsoft/superbenchmark/pull/447